### PR TITLE
Update the changelog, add the missing 2.0.0-3 versions

### PR DESCRIPTION
**Pull request by hatched**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4631/
Test PASSed.

**Comment by mitechie** [comment]
Can we drop something for the 1.8/1.9 run up to 2.0 tags in the system? Even if it's just "prepare for 2.0 release" or the like so that it's complete please?

**Comment by hatched** [comment]
We never actually cut any releases for those so that's why I left them out. Should this start including the alpha/beta/rc releases as well?

File: CHANGES.yaml [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+    - >
+      (FIX) The removal of the insecure config option. This functionality has
+      been re-enabled for now however in the near future we will be disabling
+      it for good forcing the GUI and its websockets over a secure connection.
+      Note: The GUI and it's websockets are served over a secure connection
+      by default.
+- 2.0.1:
+    - When clicking stacked charm icons the active one now is moved to the top.
+    - >
+      (FIX) Deploying the same charm multiple times woudl generate invalid
```
Comment (huwshimi) [inline CHANGES.yaml]: would

Comment (huwshimi) [inline CHANGES.yaml]: or even "could"

**Comment by anthonydillon** [comment]
Looks good :+1:

**Comment by hatched** [comment]
Thanks for the reviews! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4640/
Test PASSed.

**Pull request merged**